zen/providers/offline.py

```
@@ -129,107 +131,94 @@ class OllamaProvider:
     async def list_models(self) -> List[str]:
         """List installed models."""
         try:
             import aiohttp
+
             async with aiohttp.ClientSession() as session:
                 async with session.get(f"{self.base_url}/api/tags") as resp:
                     if resp.status == 200:
                         data = await resp.json()
-                        return [model['name'] for model in data.get('models', [])]
+                        return [model["name"] for model in data.get("models", [])]
         except:
```

Ruff (0.14.10)

Do not use bare `except`

(E722)